core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -6007,6 +6012,26 @@ public List<SqlMonotonicity> getColumnMonotonicities() {
     }
   }
 
+  private RexNode getSubqueryRexNode(SqlKind kind,
```

Comment:
I find the name of this function not very easy to understand, perhaps adding some JavaDoc will help.